### ISSUE TITLE
Could you add UDP?

### ISSUE THREAD

**Issue by HilkoLantinga** [moll/node-mitm#2]
I'm looking for an UDP (dgram) stub, this package sounds perfect, it just does not do UDP. ;)

**Comment by nicolaspanel** [comment]
hi @moll, is it still in your roadmap?

**Comment by moll** [comment]
Absolutely. I haven't looked too much into it yet though. Do you have any familiarity with Node's UDP internals?

**Comment by nicolaspanel** [comment]
No, actually I'm using an external c++ library that use UDP internally.
I've heard about [dgram](https://nodejs.org/api/dgram.html) but I've never used it.

**Comment by aguiraf** [comment]
It would be very useful!
